service/src/test/resources/ehr/mapper/condition/condition_1_resource_1_list.json

```diff
@@ -82,5 +61,8 @@
         {
             "text": "Burning feet description"
         }
-    ]
+    ],
```

Comment:
Were these change required for all of the condition JSON fixture or could they be left as they were?

Reply:
I have reverted them